### PR TITLE
Fix psalm issues

### PR DESCRIPTION
**Pull request by Tigrov**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: src/DDLQueryBuilder.php [hunk 1]
```diff
@@ -173,21 +175,18 @@ public function renameColumn(string $table, string $oldName, string $newName): s
      */
     public function getColumnDefinition(string $table, string $column): string
     {
-        $result = '';
         $sql = $this->schema->getTableSchema($table)?->getCreateSql();
 
         if (empty($sql)) {
-            return '';
+            throw new InvalidArgumentException("Table \"$table\" does not exist.");
```
Comment (darkdef) [inline src/DDLQueryBuilder.php:181]: It's not psalm fix. This change of behavior. Please move that to other PR.

File: src/DDLQueryBuilder.php [hunk 2]
```diff
@@ -173,21 +175,18 @@ public function renameColumn(string $table, string $oldName, string $newName): s
...
 
-        if (preg_match_all('/^\s*([`"])(.*?)\\1\s+(.*?),?$/m', $sql, $matches)) {
-            foreach ($matches[2] as $i => $c) {
-                if ($c === $column) {
-                    $result = $matches[3][$i];
-                }
-            }
+        $quotedColumn = preg_quote($column, '/');
+
+        if (preg_match("/^\s*([`\"])$quotedColumn\\1\s+(.*?),?$/m", $sql, $matches) !== 1) {
```
Comment (darkdef) [inline src/DDLQueryBuilder.php:186]: and this too

**Review by darkdef — CHANGES_REQUESTED** [review]

File: src/DDLQueryBuilder.php (hunk 1 above)
Comment (Tigrov) [reply src/DDLQueryBuilder.php:181]: The line in the change log is added

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,8 @@
 
 - Enh #312: Change property `Schema::$typeMap` to constant `Schema::TYPE_MAP` (@Tigrov)
 - Bug #314: Fix `Command::insertWithReturningPks()` method for empty values (@Tigrov)
+- Enh #319: Resolve psalm issues (@Tigrov)
```
Comment (vjik) [inline CHANGELOG.md:7]:
```suggestion
- Enh #319: Minor refactoring of `DDLQueryBuilder` and `Schema` (@Tigrov)
```

"Resolve psalm issues" is not about public interface of package and too common description of changes. Suggest replace it to refactoring...

File: src/DDLQueryBuilder.php (hunk 1 above)
Comment (vjik) [inline src/DDLQueryBuilder.php:181]: Add information about this exception to phpdoc of this method.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,8 @@
...
+- Enh #319: Refactor `DDLQueryBuilder::getColumnDefinition()` method (@Tigrov)
```
Comment (vjik) [inline CHANGELOG.md:8]:
```suggestion
- Bug #319: `DDLQueryBuilder::getColumnDefinition()` throw exception now if table or column doesn't exists (@Tigrov)
```

**Comment by darkdef** [comment]
Move refactoring to other PR

File: CHANGELOG.md (hunk 3 above)
Comment (Tigrov) [reply CHANGELOG.md:7]: Moved to another PR #320

File: src/DDLQueryBuilder.php (hunk 1 above)
Comment (Tigrov) [reply src/DDLQueryBuilder.php]: `throw` was removed

File: CHANGELOG.md (hunk 4 above)
Comment (Tigrov) [reply CHANGELOG.md]: Moved to another PR #320

**Comment by Tigrov** [comment]
> Move refactoring to other PR

Done

**Review by terabytesoftw — APPROVED** [review]

**Pull request merged**